ndt-server.go

```diff
@@ -96,9 +98,23 @@ func init() {
 
 // httpServer creates a new *http.Server with explicit Read and Write timeouts.
 func httpServer(addr string, handler http.Handler) *http.Server {
+	tlsconf := &tls.Config{}
+	// "modern" and "intermediate" are defined here:
+	// https://wiki.mozilla.org/Security/Server_Side_TLS
```

Comment:
This comment is referring to the previous version of the code, could you please remove it?

Reply:
Oops, yes, removed in https://github.com/m-lab/ndt-server/pull/333/commits/3ede826aff4221c65ef1b91d9d15e614201b31b7